### PR TITLE
Add schema registry to docker-compose-env file in demo

### PR DESCRIPTION
**Pull request by mpac123**

### CODE REVIEW

File: demo/docker/docker-compose-env.yml [hunk 1]
```diff
@@ -60,6 +60,15 @@ services:
 #      - zookeeper
 #      - kafka
 
+  schema-registry:
```
Comment (lciolecki) [inline demo/docker/docker-compose-env.yml:63]: Please do: schemaregistry

File: demo/docker/docker-compose-env.yml [hunk 2]
```diff
@@ -60,6 +60,15 @@ services:
...
+    container_name: nussknacker_schema-registry
```
Comment (lciolecki) [inline demo/docker/docker-compose-env.yml:64]: nussknacker_schemaregistry

File: demo/docker/docker-compose-env.yml (hunk 1 above)
Comment (mpac123) [reply demo/docker/docker-compose-env.yml]: done!

File: demo/docker/docker-compose-env.yml (hunk 2 above)
Comment (mpac123) [reply demo/docker/docker-compose-env.yml]: and also done :)

**Comment by lciolecki** [comment]
And one more.. :) base-application.conf:
```
schemaRegistryUrl: "http://schema-registry:8081"

We should figure how to use SR from IDEA and flink containter.. I think, we should have another url  for Nussknacker GUI.

**Comment by mpac123** [comment]
> And one more.. :) base-application.conf:
> 
> ```
> schemaRegistryUrl: "http://schema-registry:8081"
> ```
> 
> It's time when we should figure how to use SR from local nk and flink containter.. - not it doesnt' work.. I think, we should have another url for Nussknacker GUI.

By 'it's time' do you mean this PR? Because I'd say it's the thing to be fixed together with broken avro sources - wat do you think? I mean, for now I won't be even able to check if whatever solution I come up with works

**Pull request merged**